packages/clerk-js/src/core/clerk.ts

```diff
@@ -139,8 +140,8 @@ const defaultOptions: ClerkOptions = {
   isSatellite: false,
   signInUrl: undefined,
   signUpUrl: undefined,
-  afterSignInUrl: undefined,
-  afterSignUpUrl: undefined,
+  afterSignInUrl: '/',
+  afterSignUpUrl: '/',
```

Comment:
There are more places in our codebase where we use `displayConfig.afterSignInUrl` and `displayConfig.afterSignUpUrl`. Are those also considered?

Reply:
After the changes I made the default values will be directly taken from the `[defaultOptions](https://github.com/clerk/javascript/pull/2020/files#diff-1952d6b8ac4b5495e1dba57c32bff9ce72d43f83595c1bc8532a76cf31feb753L135-L136)` so the `displayConfig.afterSignInUrl` or `displayConfig.afterSignUpUrl` are never used.